docs/en/writing-running-appium/caps.md

```diff
@@ -101,3 +101,13 @@
 |----|-----------|-------|
 |`youiEngineAppAddress`| The IP address of the device on which the app is running. Use `localhost` for simulator.
 Use device’s IP address for a real device. |e.g. `localhost` or `192.168.1.203`|
+
+### WinAppDriver Only
+
+<expand_table>
+
+|Capability|Description|Values|
+|----|-----------|-------|
+|`platformName`| Which platform the test is being done on |e.g. `Windows`|
+|`deviceName`| The name of the device being tested on | e.g. `WindowsPC` |
+|`app`| appID of the windows app for testing or the path to the .exe file | e.g. `.../calculator.exe` |
```

Comment:
I'm wondering if this is accurate. I thought 'app' needed to be a unique crazy ID string, not just the path (yet, anyway---that's what we should do eventually). Maybe we should include the actual example for calculator, with a link to the docs on how to find this crazy unique ID?